ceph.spec.in

```diff
@@ -536,6 +577,21 @@ librbd1 or libcephfs1 instead.
 
 %endif
 
+%package compat
```

Comment:
Thanks @branto1 , good catch. "ceph-compat" came from my ancient version of this change, before we decided to convert "ceph" into a meta-package, and it can be dropped in this latest version. @alimaredia please just remove the ceph-compat subpackage.